### ISSUE TITLE
Feature / RFC - Use the body of heading if the :DESCRIPTION: drawer is not present.

### ISSUE THREAD

**Issue by j-martin** [glittershark/org-clubhouse#22]
When using, `org-clubhouse-create-story`, it uses `org-clubhouse-find-description-drawer`, as its name suggest, currently checks only if there is a drawer called `:DESCRIPTION:`. This works well enough, if the drawer exists, but it's a bit confusing the header was created out of `org-clubhouse-mode`. 

As a new user or a user trying to create new stories from existing headings, I would expect that the body of the header would be used for the description.

The `:DESCRIPTION:` make sense since it keeps things tidier and prevents random notes to the org file from being pushed to the story. We could add a variable to preserve the current behavior **AND** add a logic to pickup the body of the heading **IF** the `:DESCRIPTION:` drawer does not exist.

@glittershark what do you think?

**Comment by glittershark** [comment]
Yeah, as you said I did it this way since I like to leave all sorts of notes (including but not limited to things like LOGBOOK) under headlines that I don't necessarily want inside of clubhouse. There's a balance here between proliferation of config and trying to suit everyone's expected workflow, but after thinking about it for a couple days I think I'd be fine with a configuration option (default off) to use the body of a headline as the description of created stories if the `DESCRIPTION` drawer is absent.
